scripts/build-data.js

```diff
@@ -7,9 +7,36 @@ const flatten = require("lodash/flatten");
 const flattenDeep = require("lodash/flattenDeep");
 const mapValues = require("lodash/mapValues");
 const pickBy = require("lodash/pickBy");
+const electronToChromiumVersions = require("electron-to-chromium").versions;
 const pluginFeatures = require("../data/plugin-features");
 const builtInFeatures = require("../data/built-in-features");
 
+const electronToChromiumKeys = Object.keys(electronToChromiumVersions).reverse();
+
+const chromiumToElectronMap = electronToChromiumKeys.reduce(
+  (all, electron) => {
+    all[electronToChromiumVersions[electron]] = +electron;
+    return all;
+  }
+, {});
+const chromiumToElectronVersions = Object.keys(chromiumToElectronMap);
+
+const findClosestElectronVersion = (targetVersion) => {
+  const chromiumVersionsLength = chromiumToElectronVersions.length;
+  const maxChromium = +chromiumToElectronVersions[chromiumVersionsLength - 1];
+  if (targetVersion > maxChromium) return null;
+  const closestChrome = chromiumToElectronVersions.reduce((result, version) => {
```

Comment:
This results an incorrect version of electron being mapped, for example for async:

```json
{
  "chrome": 56,
  "electron": 1.5
}
```

Electron should be `1.6`.

Should be able to reduce (hehe) the reduce with something like:

```js
const closestChrome = chromiumToElectronVersions.find((version) => targetVersion <= version);
```

**EDIT:** even better, I guess we can drop `findClosestElectronVersion` completely and use `chromiumToElectron` [once it lands](https://github.com/Kilian/electron-to-chromium/pull/8)?